nerdlandbot/translations/Translations.csv

```diff
@@ -1,131 +1,136 @@
-"poll_start","<@{0}> is starting a poll with the following question:\n\n**{1}?**\n","<@{0}> is een peiling begonnen met de volgende vraag:\n\n**{1}?**\n"
-"poll_yes_no","{0} - Yes\n{1} - No","{0} - Ja\n{1} - Nee"
-"poll_results","{0} **The poll of  <@{1}> has ended**\n*{2}?*\n\n","{0} **De peiling van <@{1}> is afgelopen**\n*{2}*?\n\n"
-"poll_votes","{0} - {1}\n","{0} - {1}\n"
-"poll_usage","<question>?<timeout(minutes)> [answer 1];[answer 2];...;[answer 9]","<vraag>?<timeout (minuten)> [antwoord 1];[antwoord 2];...;[antwoord 9]"
-"poll_brief","Put a poll out for people to vote","Maak een peiling waarin mensen kunnen stemmen"
-"poll_help","Start a poll, with minimal arguments this will be a yes/no poll, if you provide options seperated by semicolons you can add up to 9 options.","Start een peiling, met minimale opties wordt dit een Ja/Nee peiling die <timeout> minuten duurt. Indien je zelf opties meegeeft door puntkomma gescheiden kunnen dat er maximaal 9 zijn."
-"empty_wombat_list","No wombat pictures were found...","Er werden geen foto's van wombats gevonden..."
-"add_purger_usage","<Channel ID> <Max message age (in days)>","<Kanaal ID> <Max leeftijd bericht (in dagen)>"
-"add_purger_help","**admin-only** \n \u2003 Automatically remove messages older than the configured amount of days in the channel.","**admin-functie** \n \u2003 Verwijder automatisch berichten die ouder zijn dan het opgegeven aantal dagen in het kanaal."
-"remove_purger_usage","<Channel ID>","<Kanaal ID>"
-"remove_purger_help","**admin-only** \n \u2003 Stop the automatic removal of messages in the channel.","**admin-functie** \n \u2003 Stop het automatisch verwijderen van berichten in het kanaal."
-"list_purger_help","List all channels with automatic message removal.","Lijst alle kanalen op waar berichten automatisch worden verwijderd."
-"purger_permissions","It seems I don't have permission to remove messages in this channel.","Het ziet er naar uit dat ik geen toestemming heb om berichten te verwijderen in dit kanaal."
-"purger_added","Automatically removing messages older than {1} day(s) in <#{0}>.","Vanaf nu worden berichten ouder dan {1} dag(en) automatisch verwijderd in <#{0}>."
-"purger_exists","Messages already automatically get removed in <#{0}>.","Berichten worden al automatisch verwijderd in <#{0}>."
-"purger_removed","No longer automatically removing messages in <#{0}>.","Berichten worden niet meer automatisch verwijderd in <#{0}>."
-"purger_no_exists","There's no automatic message removal in <#{0}>.","Er worden geen berichten verwijderd in <#{0}>."
-"purger_list_title","List of channels with automatic removal of messages and max age:","Lijst van kanalen waar berichten automatisch worden verwijderd en de maximale leeftijd:"
-"purger_list_item","  - <#{0}>: {1} day(s)","  - <#{0}>: {1} dag(en)"
-"channel_is_voice","You should provide a text channel, not a voice channel","Geef een tekst kanaal in plaats van een voice kanaal, foemp!"
+"Key,""en"",""nl""";;;;;;
```

Comment:
I am not sure what you did to this file, but the whole comma separation is messed up and I don't think translations will work if you implement it like this.

Reply:
Uh oh, I opened it in excel. I'll look into it and add the new issue link with the tagged message.

Reply:
Should technically word...
Probably a configuration issue with excel, not using the same format as our csv

You could try to fix that, but it will probably not be worth the trouble to do that. Simply using a text editor might be easier

Reply:
Yeah I'll just open it in VScode from now on. I was on my laptop on the couch and got lazy :)